lib/analyzer/image-inspector.ts

```diff
@@ -89,6 +89,11 @@ async function pullWithDockerBinary(
     ) {
       throw err;
     }
+
+    if (err.stderr && err.stderr.includes("invalid reference format")) {
```

Comment:
Could this be `if (err.stderr?.includes......` instead?

Reply:
As it is now, it's consistent with the rest of the error checks I'd say... Wdyt?

Reply:
No strong feeling :)